libs/forklift_inventory.py

```
@@ -8,9 +10,61 @@
+        Provider.ProviderType.OVA: OvaForkliftInventory,
+        Provider.ProviderType.RHV: OvirtForkliftInventory,
+        Provider.ProviderType.VSPHERE: VsphereForkliftInventory,
+        Provider.ProviderType.OPENSHIFT: OpenshiftForkliftInventory,
+        Provider.ProviderType.OPENSTACK: OpenstackForliftinventory,
+    })
+
+
+def create_forklift_inventory(
+    client: DynamicClient,
+    mtv_namespace: str,
+    provider: "BaseProvider",
```

Ruff (0.15.20)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)